### DISCUSSION TITLE
Is heat network optimisation really done with AI, or rather classical mathematical modelling?

### DISCUSSION THREAD

**Discussion by esc-data-science** [alan-turing-institute/ADViCE#94] [Key sector: Built Environment]
Is heat network optimisation really done with AI, or rather classical mathematical modelling? I imagine its hard to obtain large datasets on that?

**Comment by samyoung-dsci** [comment]
There's a mix I think, for two reasons. Firstly, improved accuracy of forecasts can be critical for network optimisation, and there are some cases where AI can really help with that, either by making unstructured data more structured so it can be used for forecasting, or by using AI directly for the forecasting. 
 
And secondly, there are a few people starting to explore using Reinforcement Learning rather than traditional mathematical optimisation for the overall optimisation. As I understand it, things aren't very mature there yet though.

**Comment by samyoung-dsci** [comment]
There is obviously an overlap with what is also mathematical models used in optimisation, and even what was called operations research. What is perhaps more traditionally called AI/Machine learning will be more adaptive, or for situations where the optimisation is too complicated to solve with traditional optimisers (Quasi-Newton) and perhaps more towards reinforcement learning, and evolutionary algorithms (simulated Annealing, genetic algorithms)
